src/fontinfo.rs

```diff
@@ -332,11 +333,15 @@ impl FontInfo {
     pub fn from_file<P: AsRef<Path>>(
         path: P,
         format_version: FormatVersion,
+        lib: &mut Option<Plist>,
```

Comment:
I think we should always pass `Option<&T>` instead of `&Option<T>` unless we *explicilty* want to do something like assign a new value if there's `None`, or replace some value *with* `None`.